### PR TITLE
Install Stepup SAML bundle

### PR DESCRIPTION
**Pull request by MKodde**
The installation includes the configuration of the bundle for IdP use.

For now some todo's remain:
1. Do not use the default cert+private key from the SAML bundle
2. Move configuration parameters to a more easily overwritable
paramteres.yaml file
3. Provide some test coverage

### CODE REVIEW

File: config/packages/framework.yaml [hunk 1]
```diff
@@ -9,7 +9,9 @@ framework:
         handler_id: null
         cookie_secure: auto
         cookie_samesite: lax
-
+    templating:
```
Comment (sgomez) [inline config/packages/framework.yaml:12]: This is deprecated: [UPGRADE 4.3](https://github.com/symfony/symfony/blob/master/UPGRADE-4.3.md#frameworkbundle)

File: config/packages/surfnet_saml.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+parameters:
```
Comment (sgomez) [inline config/packages/surfnet_saml.yml]: How this is only to development environments. Maybe the best location is inside `config/packages/dev` directory.

File: config/packages/framework.yaml (hunk 1 above)
Comment (MKodde) [reply config/packages/framework.yaml:12]: I know, but the stepup saml bundle still requires this as a feature. It's on the agenda to address this, but will have to be fixed in the bundle.

File: config/packages/surfnet_saml.yml (hunk 2 above)
Comment (MKodde) [reply config/packages/surfnet_saml.yml]: Yes, good point. No need to have those test params in that global config file.

File: config/packages/framework.yaml (hunk 1 above)
Comment (sgomez) [reply config/packages/framework.yaml:12]: Fortunately we have used symfony 4.4. :-)

**Review by sgomez — APPROVED** [review]

**Review by sgomez — CHANGES_REQUESTED** [review]
I realized that you are using deptrac, so Controller must be located in the Infrastructure folder

**Review by sgomez — APPROVED** [review]

**Pull request merged**